articles/010_why_ros2.md

```diff
@@ -6,6 +6,8 @@ abstract:
   This article captures the reasons for making breaking changes to the ROS API, hence the 2.0.
 published: true
 author: Brian Gerkey
+written: 2014
+last_modified: 2015
```

Comment:
I'd prefer if this data at least included the month.  I know for articles in 2014 and 2015 that doesn't matter too much, but I think for more recent articles it will be more important.

Reply:
sounds good to me! Added in 546be93f2378c51068017352041f53a31542c770

Reply:
hope the YYYY-MM format is acceptable :stuck_out_tongue:

Reply:
Yeah, that looks good to me.